analyzer/src/funTest/kotlin/managers/SbtFunTest.kt

```diff
@@ -38,11 +36,9 @@ class SbtFunTest : StringSpec({
         // Clean any previously generated POM files / target directories.
         Git().run(definitionFile.parentFile, "clean", "-fd")
 
-        val ortResult = Analyzer(AnalyzerConfiguration()).run {
-            analyze(findManagedFiles(definitionFile.parentFile, setOf(Sbt.Factory())))
-        }
+        val ortResult = analyze(definitionFile.parentFile, packageManagers = setOf(Sbt.Factory()))
 
-        patchActualResult(ortResult.withResolvedScopes(), patchStartAndEndTime = true) shouldBe expectedResult
+        patchActualResult(ortResult, patchStartAndEndTime = true) shouldBe expectedResult
```

Comment:
Shouldn't `.withResolvedScopes()` be kept here?

Reply:
I forgot to put it into the `analyze()` function. Now it's in.